stimpacks/aws/login.go

```diff
@@ -108,6 +108,11 @@ func (a *Aws) Login() error {
 			a.log.Debug("Setting {} credentials as default", profileName)
 		}
 		a.aws.SaveProfile(profileName, &profile, defaultProfile, &stimProfile)
+
+		a.log.Info("Saved {} in ~/.aws/credentials", profileName)
```

Comment:
I believe you can use the method `a.aws.GetCredentialPath()` to get the actual path that is being written to (it should normally be what you have)

Reply:
I originally did that, but then I had an an extraneous `err` I had to receive and handle for no reason (because the operation obviously succeeded if we're there).  Felt wrong, but again, perhaps this feeling is just due to my golang ignorance.

Reply:
Yeah, so code that lives in the `pkg` directory is considered third-party/consumable and doesn't presume to know how to handle errors so any code that could potentially throw errors are intentionally bubbled back to the caller. In this case, the detection of a home directory can throw an error in rare cases which is why that method will return an err as well as a the credential path.

Reply:
I do see what you're saying though - it might be nice if that class "saved" off the path that it wrote to so that it can be cached/referenced as a class attribute in later code rather than re-detecting the path.